go/app/main.go

```diff
@@ -79,33 +107,43 @@ func addItem(c echo.Context) error {
 	}
 
 	//Create new image name with sha256
-    newImageName := fmt.Sprintf("%x%s", sha256.Sum256(imageData), ".jpg")
+	newImageName := fmt.Sprintf("%x%s", sha256.Sum256(imageData), ".jpg")
 
 	//Message
 	c.Logger().Infof("We recived a %s from category: %s", name, category)
 	message := fmt.Sprintf("We recived a %s from category: %s", name, category)
 	res := Response{Message: message}
-	
-	//Create new item
-	newItem := Item{}
-	newItem.Name = name
-	newItem.Category = category
-	newItem.Image = newImageName
-
-	items, err := dataJson()
+
+	prepareDB()
+	database, err := sql.Open("sqlite3", "mercari.db")
 	if err != nil {
-		fmt.Println(err)
+		log.Fatal(err)
+	}
+
+	//Insert the data into the database
+	statement, _ := database.Prepare("INSERT INTO `Items` (`id`, `name`, `category_id`, `image_filename`) VALUES (?, ?, ?, ?);")
+	if err != nil {
+		log.Fatal(err)
 	}
+	defer statement.Close()
 
-	//Add new 
-	items.Items = append(items.Items, newItem)
+	//Getting the id corresponding to the last item
+	var itemID int
+	err = database.QueryRow("SELECT id FROM Items ORDER BY id DESC LIMIT 1").Scan(&itemID)
+	if err != nil {
+		log.Fatal(err)
+	}
```

Comment:
OPTIONAL:
FYI, some database has feature `autoincrement` and it is convenient for creating unique integer id.
In sqlite, if you create table with `INTEGER PRIMARY KEY`, and omit `id` from insertion statement, you can use this feature.
see 1. Summary of https://www.sqlite.org/autoinc.html